packages/store/src/StoreCore.sol

```diff
@@ -1086,6 +1088,7 @@ library StoreCoreInternal {
       emit StoreCore.Store_SpliceDynamicData({
         tableId: tableId,
         keyTuple: keyTuple,
+        dynamicFieldIndex: dynamicFieldIndex,
```

Comment:
Do we also need to include `startWithinField`?

Reply:
Oh good point - because `dynamicFieldIndex` only tells you which field is being updated, but not the element. I will add `startWithinField` and we can see the gas impact.

Reply:
Added it!

Reply:
looks like ~4-5% increase in dynamic field operations